front/components/QuickStartGuide.tsx

```diff
@@ -0,0 +1,207 @@
+  show,
+  onClose,
+}: {
+  show: boolean;
+  onClose: () => void;
+}) {
+  return (
+    <Modal
+      isOpen={show}
+      variant="side-md"
+      hasChanged={false}
+      onClose={onClose}
+      title="🚀 Getting started with Dust"
+    >
+      <div className="flex h-full flex-col">
+        <ScrollArea className="flex-1">
+          <div className="p-5">
+            <div className="w-full">
+              <div className="relative w-full overflow-hidden rounded-lg pb-[56.20%]">
+                <iframe
+                  src="https://fast.wistia.net/embed/iframe/v90n8beuh9?seo=true&videoFoam=false"
+                  title="Dust product tour"
+                  allow="autoplay; fullscreen"
+                  className="absolute left-0 top-0 h-full w-full rounded-lg"
+                />
+              </div>
+            </div>
+
+            <div className="mt-8 space-y-8">
+              <div>
+                <h5 className="text-lg font-semibold">
+                  👋 Hello <span className="text-success-500">@assistants</span>
+                </h5>
+                <CardButton variant="secondary">
+                  <div className="flex flex-row space-y-4">
+                    <div className="flex flex-col space-y-4">
+                      <p>
+                        In Dust, you have access to{" "}
+                        <strong>multiple AI&nbsp;assistants</strong>.
+                      </p>
+                      <p className="text-sm">
+                        You can call an assistant by typing <strong>"@"</strong>{" "}
+                        and the name of the assistant. You can even call several
+                        assistants at the same time or chain them in one
+                        conversation.
+                      </p>
+                    </div>
+                    <div>
+                      <img src="/static/quick_start_guide_input_bar.png" />
+                    </div>
+                  </div>
+                </CardButton>
+              </div>
+
+              <div className="grid grid-cols-2 gap-4">
+                <div>
+                  <h5 className="mb-2 text-lg font-semibold">
+                    👩‍🎨🦸‍♀️🥷🧑‍🚀 Multiple Assistants
+                  </h5>
+                  <CardButton variant="secondary">
+                    <div className="space-y-4">
+                      <p className="font-bold">
+                        Specialized assistants can help answer questions, really
+                        well.
+                      </p>
+                      <div className="space-y-2 text-sm">
+                        <p>
+                          Assistants in Dust can be provided by Dust, by your
+                          company (Company assistants) or by your coworkers
+                          (Shared assistants).
+                        </p>
+                        <p>
+                          Specialised assistants generally combine a model with
```

Comment:
OOC: what's the policy between using US vs UK English? (applies to specialized and summarize in this file)

Reply:
🤷🏼‍♂️

Reply:
current code is using US , so should change here to `Specialized` and `summarize`